homework/homework_part_2/javascript/app.js

```diff
@@ -0,0 +1,103 @@
+//1. With interpolation you apply your variable inside 
+// of a string and concatenation you write everything out 
+// as a string 
+// INTERPOLATION: const red = 2
+//const blue = 5
+//console.log(`Hello World ${red + blue}`)
+//CONCATENATION: console.log("Hello World" +  " AND GOOD DAY");
+
+//2. DRY stands for Don't Repeat Yourself, we should pay 
+// attention to it because we don't want to repeat
+//code that is almost identical to eachother when 
+//we can condense it down
+
+//3. you declare a variable at the top ex: let g =4;
+// you assign when you change the value of in this case g 
+// g = 7;
+// and it can be defined depending where the declaration sits
+// if your decleration is outside a for loop and your console.log
+// is inside it will be undefined
+
+//4. if you don't have to reassign variable use const
+// because it will stand out and you'll know exactly 
+// what it is, where as let can be reassigned and used 
+// in loops that may have variable changes
+
+//5. A parent directory is a higher directory that holds
+// other files/folders "children directories" inside it
+
+//6. if you want to know more about git you could use man git
+
+//7. tab completion is awesome because it finishes your line
+// for you and makes it so you avoid misspelling and its faster!
+
+   const a = 4;
+   const b = 53;
+   const c = 57;
+   const d = 16;
+   const e = 'Kevin';
+  
+
+ console.log(a < b );
+ console.log(c > d )
+ console.log('Name' == 'Name');
+ console.log(a < b < c );
+ console.log(a != a < c );
```

Comment:
again, what does this mean?  this would reduce to `false < 57`.  That doesn't really make sense.  Check out the values of a and c and see if you can come up with a simpler, less arcane answer.